#identificaci√≥n del individuo seleccionado
string user_name
string id_experiment
int32 num_ind

float64[] emotion_value #valence, arousal
int32 storage_pos
